lib/Doctrine/ORM/AbstractQuery.php

```diff
@@ -395,14 +362,12 @@ public function setParameter($key, $value, $type = null)
     /**
      * Processes an individual parameter value.
      *
-     * @param mixed $value
-     *
      * @return mixed[]|string|int|float|bool|object|null
      * @psalm-return array|scalar|object|null
```

Comment:
Do these annotations provide any real value? They're hardly more specifiy than `mixed`.

Reply:
I'll try widening to `mixed` on 2.13.x, see if that impacts static analysis.

Reply:
https://github.com/doctrine/orm/pull/9801